presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -435,31 +446,31 @@ public double getSampleRatio(SampledRelation relation)
         return sampleRatios.get(relation);
     }
 
-    public static class JoinInPredicates
+    public static class JoinSideExpressions
```

Comment:
Is it "join side" expressions? Or join "side expressions"? I'm trying to understand what this class stands for.